CellProfiler.py

```diff
@@ -0,0 +1,3 @@
+from cellprofiler import __main__
+
+__main__.main()
```

Comment:
We should use:

```Python
if __name__ == "__main__":
    cellprofiler.__main__.main()
```

per [docs.python.org](https://docs.python.org/2/library/__main__.html).